cms/templates/emails/utilities_bulk_update_done_message.txt

```diff
@@ -0,0 +1,15 @@
+<%! from django.utils.translation import ugettext as _ %>
+${_("Hello, {username}").format(username=username)}
+
+% if success:
+${_("Your bulk update on the {course_name} course has successfully completed. Previously published problems have been automatically republished.").format(course_name=course_name)}
+
+${_("The following settings were modified:")}
+
+% for key, value in modified_settings.iteritems():
+${_("{key}: {value}").format(key=key, value=value)}
```

Comment:
No need to translate this string as there isn;t anything to actually translate...